mono/metadata/threadpool-worker-default.c

```diff
@@ -682,9 +705,14 @@ monitor_thread (gpointer unused)
 
 		g_assert (worker.monitor_status != MONITOR_STATUS_NOT_RUNNING);
 
-		// counter = COUNTER_READ ();
-		// printf ("monitor_thread: starting = %d working = %d parked = %d max_working = %d\n",
-		// 	counter._.starting, counter._.working, counter._.parked, counter._.max_working);
+#if 0
```

Comment:
disabled/remove or enabled/keep?

Reply:
It's very noisy - the monitor thread gets here about a dozen times per second - I was worried that it would drown out other debug info.

Reply:
OK, maybe we could add that comment to explain why we have "dead" code.